src/guide/typescript-support.md

```diff
@@ -372,6 +372,84 @@ year.value = 2020 // ok!
 If the type of the generic is unknown, it's recommended to cast `ref` to `Ref<T>`.
 :::
 
+### Typing Template Refs
+
+Sometimes you might need to annotate a template ref for a child component in order to call its public method. For example, we have a `MyModal` child component with a method that opens the modal:
+
+```ts
+import { defineComponent, ref } from 'vue'
+
+const MyModal = defineComponent({
+  setup() {
+    const isContentShown = ref(false)
+    const open = () => (isContentShown.value = true)
+
+    return {
+      isContentShown,
+      open
+    }
+  }
+})
+```
+
+We want to call this method via a template ref from the parent component:
+
+```html
+<template>
+  <button @click="openModal">Open from parent</button>
+  <my-modal ref="modal" />
+</template>
+```
```

Comment:
I think we can remove this code block, it seems to be repeating some of the code from the next code block.

Reply:
Sorry, totally forgot to remove this one. Should be fine now!